pkg/provisioner/controller.go

```diff
@@ -283,7 +281,7 @@ func (c *obcController) handleProvisionClaim(key string, obc *v1alpha1.ObjectBuc
 	// It is assumed that if the get claim fails, no resources were generated to begin with.
 	defer func() {
 		if err != nil {
-			log.Error(err, "cleaning up reconcile artifacts")
+			log.Info("provisioning errored, cleaning up API resources")
 			if !pErr.IsBucketExists(err) && ob != nil && isDynamicProvisioning {
```

Comment:
maybe add comment that the ob is non-nil after call to `Provision` or `Revoke` and *before* any k8s resources have been created. I forgot this and spent a little time thinking about error cases where secret or cm have been created before the ob...

Reply:
+1